### DISCUSSION TITLE
Examples: JSON Tree broken

### DISCUSSION THREAD

**Discussion by lovetox** [Textualize/textual#1769] [Q&A] [answered]
Hi,

the json_tree.py example seems broken with 0.10.1

i get an import error

```python
Traceback (most recent call last):
  File "/home/lovetox/projects/testtextualapp/textual/examples/json_tree.py", line 8, in <module>
    from textual.widgets.tree import TreeNode
ModuleNotFoundError: No module named 'textual.widgets.tree'
```

**Comment by davep** [comment, marked as answer]
Where did you get the `jason_tree.py` example from? By the looks of things you're attempting to use the code from the `main` branch of Textual, with the library as of v0.10.1.  [The way that `TreeNode` is imported has changed](https://github.com/Textualize/textual/blob/main/CHANGELOG.md#changed) so mixing and matching that code won't work. If you want to use the examples that work with v0.10.1 it's best to download either [download the source for that release](https://github.com/Textualize/textual/releases/tag/v0.10.1), or [use the example from the v0.10.1 tag](https://github.com/Textualize/textual/blob/v0.10.1/examples/json_tree.py).
